api-reference/docstubs/api/calendar-getschedule.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "getSchedule"
```

Comment:
Global: Be sure to apply the correct title convention for function and action topics.

Reply:
Good spot.